src/sbt-test/cucumber/scenario_outlines/build.sbt

```diff
@@ -0,0 +1,19 @@
+name := "cucumber-test"
+
+organization := "com.waioeka.sbt"
+
+version := "0.0.4"
+
+scalaVersion := "2.12.2"
+
+libraryDependencies ++= Seq (
+        "org.scalatest" %% "scalatest" % "3.0.1" % "test",
+        "io.cucumber" % "cucumber-core" % "2.0.0" % "test",
+        "io.cucumber" %% "cucumber-scala" % "2.0.0" % "test",

```

Comment:
This is a rather old version of latest is 4.7.1

https://search.maven.org/search?q=g:%22io.cucumber%22%20AND%20a:%22cucumber-scala_2.12%22